### PR TITLE
[GITHUB-1] Install and configure RabbitMQ

### PR DESCRIPTION
**Pull request by wojtek-oledzki**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+rabbitmq:
+  cacert: "files/rabbitmq_cacert.pem"
+  env: false
+  new_only: 'no'
+  server_cert: "files/rabbitmq_server_cert.pem"
+  server_key: "files/rabbitmq_server_key.pem"
+  ssl: false
+
+  conf:
+      tcp:
```
Comment (wojtek-oledzki) [inline defaults/main.yml]: indentation :(

**Comment by fdupoux** [comment]
There is still a reference to ansible-city:
rm -rf tests/ansible-city.*

**Comment by wojtek-oledzki** [comment]
@fdupoux @lobsterdore fixed and cleaned up a bit

File: meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+
+allow_duplicates: yes
```
Comment (lobsterdore) [inline meta/main.yml]: Do we want to allow duplicates for this role?

Comment (wojtek-oledzki) [inline meta/main.yml]: nope - removing

**Comment by lobsterdore** [comment]
LGTM :+1:

**Comment by fdupoux** [comment]
Looks good to me

**Pull request merged**